runtime/lua/vim/_system.lua

```diff
@@ -245,7 +245,13 @@ local function spawn(cmd, opts, on_exit, on_error)
   local handle, pid_or_err = uv.spawn(cmd, opts, on_exit)
   if not handle then
     on_error()
-    error(('%s: "%s"'):format(pid_or_err, cmd))
+    if opts.cwd and not uv.fs_stat(opts.cwd) then
+      error(("%s: '%s'"):format(pid_or_err, opts.cwd))
```

Comment:
to match the one below, use " quotes in the message. also should we add a `cwd` hint in the message?
```suggestion
      error(('%s: cwd: "%s"'):format(pid_or_err, opts.cwd))
```

Reply:
How about we use `'` quote for both? I think they are usually preferred for string literals?

Reply:
sure, up to you.

historically, i have been using " for user-facing messages, since it is more visually noticeable. otoh, yes, we prefer single-quotes in Lua, but i consider that separate from user messages (where often, or usually, quotes are not syntax but just delimiters).

not a blocker